### PR TITLE
feat: init repo

### PR DESCRIPTION
**Pull request by FoseFx**
I realize this a massive PR, so here are some pointers (also I recommend to check it out locally):
there are `/libs` and `/services`, the latter only has the new `user-svc` service.

In `libs` there are
*  `common` - the library for everything related to setup and dapr
* `logging` - zerolog helpers
* `hwgorm` - gorm helpers
* `hwutil` - general purpose utility library

### CODE REVIEW

**Comment by JannikWibker** [comment]
image

any particular reason for this?, this seems like a rather long list

**Comment by FoseFx** [comment]
copy & paste from hw/rest-api

**Comment by JannikWibker** [comment]
ahh true
then the question becomes why hw/rest-api had to have gitignore entries for vim, emacs, dart, flutter, android, pycharm, react, react native, python, xcode, node, cordova, vue, etc.

**Comment by MaxSchaefer** [comment]
The user-svc is a nice example for now, but in the futures the organization-part should be handled by a separate service.

**Comment by FoseFx** [comment]
As of today's meeting we scrap Auth v1 anyway (in favor of a IAM solution, Keycloak?), so this service will probably experience heavy changes anyway.

**Comment by MaxSchaefer** [comment]
> As per today's meeting we scrap Auth v1 anyway (in favor of a IAM solution, Keycloak?), so this service will probably experience heavy changes anyway.

Yes, Keycloak was the plan.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,1569 @@
+# Created by https://www.toptal.com/developers/gitignore/api/vim,emacs,dart,flutter,android,androidstudio,intellij,intellij+all,intellij+iml,pycharm,pycharm+all,pycharm+iml,go,goland,goland+iml,goland+all,visualstudiocode,react,reactnative,python,pythonvanilla,xcode,xcodeinjection,node,nativescript,cordova,apachecordova,ionic3,vue,vuejs
```
Comment (MaxSchaefer) [inline .gitignore:1]: Already discussed [here](https://github.com/helpwave/services/pull/1#issuecomment-1356375726).

File: services/user-svc/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# user-svc
+
+> User and Organization management service
```
Comment (MaxSchaefer) [inline services/user-svc/README.md:3]:
```suggestion
> User management service
```

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,481 @@
+# Created by https://www.toptal.com/developers/gitignore/api/go,vim,goland,intellij,goland+all,goland+iml,intellij+all,intellij+iml,visualstudiocode
```
Comment (MaxSchaefer) [inline .gitignore:1]: I guess not perfect but okay for now.

**Review by MaxSchaefer — APPROVED** [review]
Nice! This should unblock the other pull requests.

**Pull request merged**